cmd/src/actions_exec.go

```diff
@@ -183,7 +184,20 @@ Format of the action JSON files:
 			return errors.Wrap(err, "invalid JSON action file")
 		}
 
-		ctx := context.Background()
+		ctx, cancel := context.WithCancel(context.Background())
+		c := make(chan os.Signal, 1)
+		signal.Notify(c, os.Interrupt)
+		defer func() {
+			signal.Stop(c)
+			cancel()
```

Comment:
is there any effect of calling cancel twice?

Reply:
Why should it be called twice?

Reply:
I would have guessed once when the signal handler hits in the go function, and once when the outer function is over running the defer function? 🤔 not sure if that's the case though ..

Reply:
Ah! I see.

https://golang.org/pkg/context/#CancelFunc

> After the first call, subsequent calls to a CancelFunc do nothing.